instance/tests/integration/integration_instance.py

```diff
@@ -354,7 +354,7 @@ def assert_static_content_overrides_work(self, instance, appserver, page):
         homepage_html = get_url_contents(instance.url)
         self.assertIn(instance.static_content_overrides['homepage_overlay_html'], homepage_html)
 
-        page_url = '{}/{}'.format(instance.url, page)
+        page_url = instance.url + page
```

Comment:
Is there a particular reason for this change? Will the previous syntax not work with the Python version upgrades?

Reply:
Yes, @pcockwell, by some reason `instance.url` contains trailing slash in Juniper unlike previous release, so this was breaking test.

Reply:
@0x29a, `instance.url` had a trailing slash even on Ironwood. So fixing this is good. 👍🏼